src/parser.rs

```diff
@@ -169,6 +169,21 @@ impl<'a, D> CommandParser<'a, D> {
                 .unwrap_or(self.buffer.len())
     }
 
+    /// Finds the index of the character after the raw string parameter (comma or end of data).
+    fn find_end_of_raw_string_parameter(&self) -> usize {
+        self.buffer_index
+            + self
+                .buffer
+                .get(self.buffer_index..)
+                .map(|buffer| {
+                    buffer
+                        .iter()
+                        .take_while(|byte| **byte != b',' && !(**byte as char).is_ascii_control())
```

Comment:
I feel like this should only look at commas, not at anything else like control characters.